devel/addon/kyverno/values.yaml

```diff
@@ -0,0 +1,24 @@
+config:
+  # resource types to be skipped by kyverno policy engine
+  # Make sure to surround each entry in quotes so that it doesn't get parsed
+  # as a nested YAML list. These are joined together without spaces in the configmap
+  resourceFilters:
+  - "[Event,*,*]"
+  - "[*,kube-system,*]"
+  - "[*,kube-public,*]"
+  - "[*,kube-node-lease,*]"
+  - "[Node,*,*]"
+  - "[APIService,*,*]"
+  - "[TokenReview,*,*]"
+  - "[SubjectAccessReview,*,*]"
+  - "[SelfSubjectAccessReview,*,*]"
+  - "[*,kyverno,*]"
+  - "[Binding,*,*]"
+  - "[ReplicaSet,*,*]"
+  - "[ReportChangeRequest,*,*]"
+  - "[ClusterReportChangeRequest,*,*]"
+  # Ignore some cert-manager E2E test components that can't easily be configured
+  # to meet the pod security policy.
+  - "[*,bind,*]"
+  - "[*,ingress-nginx,*]"
+  - "[*,sample-external-issuer-system,*]"
```

Comment:
This is my alternative work around to configure Kyverno to ignore some of the E2E components. 
See https://kyverno.io/docs/installation/#resource-filters

Reply:
See https://github.com/cert-manager/sample-external-issuer/issues/18